src/index.d.ts

```diff
@@ -28,7 +28,7 @@ declare module 'react-lottie-player' {
 
       /** Lottie `AnimationItem` Instance */
       ref?: React.Ref<AnimationItem | undefined>
-    } & ({ path?: string } | { animationData?: { ['default']: object } | object })
+    } & ({ path?: string } | { assetsPath?: string } | { animationData?: { ['default']: object } | object })
```

Comment:
Not sure if this is correct typing. Doesnt this mean that you cannot specify both assetPath and path/animationData together? Is that intented?